src/transcoder/transcodedialog.cpp

```diff
@@ -118,7 +119,7 @@ TranscodeDialog::TranscodeDialog(QWidget* parent)
 }
 
 TranscodeDialog::~TranscodeDialog() {
-  delete log_ui_;
+  delete details_ui_;
```

Comment:
Can you change these to use `unique_ptr` while you're here?

Reply:
Done.